app/pages/learn/01_tutorial/07_rich_client_apps/02_introduction-javafx-animation.md

```diff
@@ -10,27 +10,24 @@ layout: learn/tutorial.html
 subheader_select: tutorials
 main_css_id: learn
 description: "Learn to create advanced JavaFX animations"
-last_update: 2024-05-17
+last_update: 2024-05-22
 ---
 
 The [javafx.animation](javafxdoc:AnimationPackageSummary) package offers a simple framework for creating animations and transitions in a JavaFX application.
-It operates on the principle of [WritableValue\<T\>](javafxdoc:WritableValue) which are used across JavaFX.
+It operates on the principle of [`WritableValue\<T\>`](javafxdoc:WritableValue), which is used across JavaFX. `WritableValue<T>` is an interface that wraps a value that can be read and set.
+It is commonly used for storing properties in JavaFX UI elements, like `width` or `height` in the [`Rectangle`](javafxdoc:Rectangle) shape.
 It additionally provides a variety of built-in transitions for common effects, support for parallel and sequential transitions, and the ability to handle events upon animation completion.
-I will go through all types of animations, starting with `Animation` and its subclasses `Transition` and `Timeline`, before representing a lower level animation with `AnimationTimer`.
+The article goes through all types of animations, starting with `Animation` and its subclasses `Transition` and `Timeline`, before representing a lower level animation with `AnimationTimer`.
 While `Transition` provides a simpler and more user-friendly way to create animations, `Timeline` offers greater flexibility and is suitable for more complex animations.
 In contrast, `AnimationTimer` is designed for frame-by-frame updates and does not make use of `WritableValue<T>`.
 
-## WritableValue
-`WritableValue<T>` is an interface that wraps a value that can be read and set. It is commonly used for storing properties in JavaFX UI elements, like
-`width` or `height` in the [Rectangle](javafxdoc:Rectangle) shape.
-
 ## Animation
 
-The [Animation](javafxdoc:Animation) abstract class provides the core functionality for `Transition` and `Timeline` animations.
+The abstract class [`Animation`](javafxdoc:Animation) provides the core functionality for `Transition` and `Timeline` animations and can't be implemented directly.
```

Comment:
> Aside from that, does it even bring much benefit to mention these classes being `abstract`?

Otherwise the reader could think that this is one way of creating an animation in javafx by creating an instance of it until they notice it is an abstract class and just the base for `Transition` and `Timeline`.